applications/nfc/scenes/nfc_scene_save_name.c

```diff
@@ -32,12 +32,12 @@ const void nfc_scene_save_name_on_enter(void* context) {
     view_dispatcher_switch_to_view(nfc->view_dispatcher, NfcViewTextInput);
 }
 
-const bool nfc_scene_save_name_on_event(void* context, SceneManagerEvent event) {
+bool nfc_scene_save_name_on_event(void* context, SceneManagerEvent event) {
     Nfc* nfc = (Nfc*)context;
 
     if(event.type == SceneManagerEventTypeCustom) {
         if(event.event == SCENE_SAVE_NAME_CUSTOM_EVENT) {
-            if(nfc->dev.dev_name) {
+            if(strcmp(nfc->dev.dev_name, "")) {
```

Comment:
@gornekich what kind of logic expected here?

Reply:
Fix is correct